.github/workflows/test-single.yml

```diff
@@ -210,6 +210,12 @@ jobs:
           mkdir ${{ github.workspace }}/.chia
           mv ${{ github.workspace }}/test-cache-${{ env.BLOCKS_AND_PLOTS_VERSION }}/* ${{ github.workspace }}/.chia
 
+      - name: Copy pre-generated plots for CLI / Simulator Tests
+        run: |
+          mkdir ${{ github.workspace }}/.chia/simulator
+          mkdir ${{ github.workspace }}/.chia/simulator/plots
+          cp ${{ github.workspace }}/.github/test-plots/* ${{ github.workspace }}/.chia/simulator/plots
```

Comment:
I see, so it looks like it's deliberate. I think it's risky to check-in large files in git. They'll stick around in the repo forever. I think we should look for an alternative.

I would propose a new command that creates the test plots and puts them in the right place. That way the simulator could be used by people installing the chia package as well, and not just ones using it from the git repo.

Reply:
test plots removed, might as well test the creation. (ci 1 min longer)